### DISCUSSION TITLE
Amazing job!

### DISCUSSION THREAD

**Discussion by felipetesc** [lapce/lapce#853] [General]
This project is amazing! Congrats!

**Comment by Goddard** [comment]
Yeah I agree.  Looks like a great option.

**Comment by Meiyo40** [comment]
I agree, this is good work, I really want to see where you go from here.

**Comment by tno2007** [comment]
And just this week I was looking for a fast code editor.
Ofcourse its early days now, but I'm happy with what you guys are doing.
Thank you!
